Tests/TestData/spy_with_rsv.csv

```diff
@@ -0,0 +1,501 @@
+10/04/2013 12:00:00 AM,167.74,169.06,167.53,168.89,96600700,0.005015610596716022
+10/07/2013 12:00:00 AM,167.43,168.45,167.25,167.43,95653200,0.005107307537323684
+10/08/2013 12:00:00 AM,167.45,167.62,165.36,165.48,176560600,0.005187040859560394
+10/09/2013 12:00:00 AM,165.8,166.2,164.53,165.6,168100500,0.005581916644070639
+10/10/2013 12:00:00 AM,167.29,169.26,167.23,169.17,193988300,0.005606227716222606
+10/11/2013 12:00:00 AM,168.9,170.32,168.77,170.26,104956400,0.005405222902240146
+10/14/2013 12:00:00 AM,169.24,171.08,169.08,170.94,111248400,0.005456117735173119
+10/15/2013 12:00:00 AM,170.52,171.15,169.47,169.7,154901700,0.005551266207356592
+10/16/2013 12:00:00 AM,170.75,172.16,170.63,172.07,160895600,0.005011254308924856
+10/17/2013 12:00:00 AM,171.37,173.32,171.34,173.22,128890400,0.004903967368899299
+10/18/2013 12:00:00 AM,173.86,174.51,173.51,174.39,138055900,0.004608634656479063
+10/21/2013 12:00:00 AM,174.45,174.75,174.01,174.4,104058600,0.004448528309500735
+10/22/2013 12:00:00 AM,174.91,175.93,174.66,175.41,126638900,0.003984496062544911
+10/23/2013 12:00:00 AM,174.83,174.89,173.96,174.57,105043800,0.004090810910046734
+10/24/2013 12:00:00 AM,174.9,175.37,174.51,175.15,70037700,0.004115490518509659
+10/25/2013 12:00:00 AM,175.51,176.0,175.17,175.95,93453900,0.003981093439543957
+10/28/2013 12:00:00 AM,175.9,176.47,175.7,176.23,84497900,0.0035074215108520535
+10/29/2013 12:00:00 AM,176.6,177.24,176.38,177.17,87200600,0.0034562226733107597
+10/30/2013 12:00:00 AM,177.4,177.51,175.66,176.29,135506600,0.003924729165769538
+10/31/2013 12:00:00 AM,176.18,176.89,175.53,175.79,133352300,0.0041690567133654925
+11/01/2013 12:00:00 AM,176.05,176.61,175.22,176.21,142771700,0.004486488966203732
+11/04/2013 12:00:00 AM,176.71,176.9,175.9825,176.83,85257800,0.004447989309512458
+11/05/2013 12:00:00 AM,176.16,176.75,175.57,176.27,85466500,0.004502005308824273
+11/06/2013 12:00:00 AM,177.03,177.5,176.54,177.17,87123100,0.004542140421944936
+11/07/2013 12:00:00 AM,177.56,177.64,174.76,174.93,148632200,0.00469812394441893
+11/08/2013 12:00:00 AM,174.85,177.31,174.85,177.29,136500100,0.004623584370357659
+11/11/2013 12:00:00 AM,177.14,177.53,176.91,177.32,66226700,0.004606863639299921
+11/12/2013 12:00:00 AM,176.95,177.36,176.37,176.96,83193500,0.0043158105487779725
+11/13/2013 12:00:00 AM,176.1,178.41,176.09,178.38,99795600,0.003944082716530641
+11/14/2013 12:00:00 AM,178.53,179.42,178.25,179.27,102965800,0.0036406782753026575
+11/15/2013 12:00:00 AM,179.57,180.12,179.33,180.05,102759600,0.0034252918715610295
+11/18/2013 12:00:00 AM,180.39,180.5,179.02,179.42,98459500,0.0033964907639071956
+11/19/2013 12:00:00 AM,179.31,179.87,178.72,179.03,93045800,0.003485301629742071
+11/20/2013 12:00:00 AM,179.41,179.93,177.98,178.47,124692900,0.0038373636953685725
+11/21/2013 12:00:00 AM,179.0,180.05,178.86,179.91,91804700,0.00394520300884539
+11/22/2013 12:00:00 AM,180.01,180.83,179.77,180.81,81234700,0.003981481670311425
```

Comment:
Just double checking, but these indicator values are really small for example 0.39%. might be worth making sure there's not some issue

Reply:
I remember using a volality threshold of 0.003 when trading using Trality. So that's quite comparable values.